torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -518,7 +521,7 @@ def rotate_bounding_box(
     angle: float,
     expand: bool = False,
     center: Optional[List[float]] = None,
-) -> torch.Tensor:
+) -> Tuple[torch.Tensor, Tuple[int, int]]:
```

Comment:
I'm not going to insist on this but it's kinda weird that sometimes we define `size` as `List[int]` and others `image_size` as `Tuple[int, int]`. I do agree that the latter would have been preferable but due to JIT we can't do it. Shall we align everywhere in a separate PR?

Reply:
> Shall we align everywhere in a separate PR?

As discussed offline, we are not sure whether we should or not. I'll open an issue soon with a proposal to drop the wrong `List[int]` from all kernels in favor of the correct annotations.

Since this is not blocking, I'm going ahead with `Tuple[int, int]` as return type and we can revisit after we have resolved the discussion in the issue to be created.